<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MellowPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mellowplayer.qrc">
    <normaloff>:/icons/mellowplayer.png</normaloff>:/icons/mellowplayer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageHome">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="0">
         <widget class="QWidget" name="background" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget#background {
	border-image: url(:/icons/home-background.png) 0 0 0 0 stretch stretch;
	border-width: 0px;
}

</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons/mellowplayer.svg&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255)</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;MellowPlayer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>60</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>40</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="HoverableButton" name="btCreatePlugin">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton , QPushButton:hover,  QPushButton:pressed, QPushButton:focus {
	background-color: transparent white;
}

QPushButton:hover,  QPushButton:pressed, QPushButton:focus {
	border: 1px #3daee9;
    border-radius: 3px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../mellowplayer.qrc">
                 <normaloff>:/icons/plugins-white.svg</normaloff>:/icons/plugins-white.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="HoverableButton" name="pushButtonSelect">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton , QPushButton:hover,  QPushButton:pressed, QPushButton:focus {
	background-color: transparent white;
}

QPushButton:hover,  QPushButton:pressed, QPushButton:focus {
	border: 1px #3daee9;
    border-radius: 3px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../mellowplayer.qrc">
                 <normaloff>:/icons/folder-music-white.svg</normaloff>:/icons/folder-music-white.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>92</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="HoverableButton" name="pushButtonPreferences">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton , QPushButton:hover,  QPushButton:pressed, QPushButton:focus {
	background-color: transparent white;
}

QPushButton:hover,  QPushButton:pressed, QPushButton:focus {
	border: 1px #3daee9;
    border-radius: 3px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../mellowplayer.qrc">
                 <normaloff>:/icons/preferences-system-white.svg</normaloff>:/icons/preferences-system-white.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelToolTip">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <stylestrategy>PreferDefault</stylestrategy>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string>Select a music streaming service</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageWeb">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WebView" name="webView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="popupUpdate" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#popupUpdate {background-color: #3daee9;};</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons/preferences-desktop-notification.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPopupUpdate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;A new version of MellowPlayer is available!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPopupOpenBrowser">
         <property name="text">
          <string>Open browser</string>
         </property>
         <property name="icon">
          <iconset theme="internet-web-browser">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPopupDetails">
         <property name="text">
          <string>Details</string>
         </property>
         <property name="icon">
          <iconset theme="text-x-changelog">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPopupClose">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="window-close">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Applicatio&amp;n</string>
    </property>
    <addaction name="actionSelect_service"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Pla&amp;yer</string>
    </property>
    <addaction name="actionPlayPause"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_to_favorites"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout_MellowPlayer"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionReport_a_bug"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>V&amp;iew</string>
    </property>
    <addaction name="actionShow_menu"/>
    <addaction name="actionFullscreen"/>
    <addaction name="actionShow_lyrics"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navi&amp;gation</string>
    </property>
    <addaction name="actionHome"/>
   </widget>
   <widget class="QMenu" name="menuDeveloper">
    <property name="title">
     <string>&amp;Developer</string>
    </property>
    <addaction name="actionCreate_plugin"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuView"/>
   <addaction name="menuDeveloper"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionSelect_service"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_to_favorites"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionPlayPause"/>
   <addaction name="actionNext"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLyrics">
   <property name="windowTitle">
    <string>&amp;Lyrics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEditLyrics">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::WidgetWidth</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSelect_service">
   <property name="icon">
    <iconset theme="folder-music" resource="../mellowplayer.qrc">
     <normaloff>:/icons/folder-music.png</normaloff>:/icons/folder-music.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Select a music streaming service</string>
   </property>
   <property name="toolTip">
    <string>Select a music streaming service</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-system" resource="../mellowplayer.qrc">
     <normaloff>:/icons/preferences-system.png</normaloff>:/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="icon">
    <iconset theme="media-playback-start" resource="../mellowplayer.qrc">
     <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Media Play</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="media-seek-forward" resource="../mellowplayer.qrc">
     <normaloff>:/icons/media-seek-forward.png</normaloff>:/icons/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Media Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="media-seek-backward" resource="../mellowplayer.qrc">
     <normaloff>:/icons/media-seek-backward.png</normaloff>:/icons/media-seek-backward.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;revious</string>
   </property>
   <property name="shortcut">
    <string>Media Previous</string>
   </property>
  </action>
  <action name="actionAbout_MellowPlayer">
   <property name="icon">
    <iconset theme="help-about" resource="../mellowplayer.qrc">
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About MellowPlayer</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionReport_a_bug">
   <property name="icon">
    <iconset theme="tools-report-bug" resource="../mellowplayer.qrc">
     <normaloff>:/icons/tools-report-bug.png</normaloff>:/icons/tools-report-bug.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Report an issue...</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit" resource="../mellowplayer.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRestoreWindow">
   <property name="icon">
    <iconset theme="view-restore" resource="../mellowplayer.qrc">
     <normaloff>:/icons/view-restore.png</normaloff>:/icons/view-restore.png</iconset>
   </property>
   <property name="text">
    <string>Restore window</string>
   </property>
   <property name="toolTip">
    <string>Restore the window</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/about-qt.png</normaloff>:/icons/about-qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionAdd_to_favorites">
   <property name="icon">
    <iconset theme="emblem-favorite" resource="../mellowplayer.qrc">
     <normaloff>:/icons/emblem-favorite.png</normaloff>:/icons/emblem-favorite.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add to favorites</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset theme="help-contents" resource="../mellowplayer.qrc">
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="go-home" resource="../mellowplayer.qrc">
     <normaloff>:/icons/go-home.png</normaloff>:/icons/go-home.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Home</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionShow_menu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="show-menu">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Show menu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-fullscreen" resource="../mellowplayer.qrc">
     <normaloff>:/icons/view-fullscreen.png</normaloff>:/icons/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionCreate_plugin">
   <property name="icon">
    <iconset theme="plugins" resource="../mellowplayer.qrc">
     <normaloff>:/icons/plugins.png</normaloff>:/icons/plugins.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create plugin...</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionShow_lyrics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-media-lyrics"/>
   </property>
   <property name="text">
    <string>Show lyrics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebView</class>
   <extends>QWidget</extends>
   <header>views/webview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HoverableButton</class>
   <extends>QPushButton</extends>
   <header>views/hoverable_button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../mellowplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonPopupClose</sender>
   <signal>clicked()</signal>
   <receiver>popupUpdate</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
